ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+* Provide a general summary of the issue in the **Title** above.  
+* Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+### Important
+Mark with [x] to select. Leave as [ ] to unselect.
+
+### When reporting a bug/issue, please be sure to include the following:
```

Comment:
This heading is pretty verbose, how about just `### When reporting a bug/issue`?